src/components/Personalization/handlers/createProcessPropositions.js

```diff
@@ -193,7 +200,7 @@ export default ({ schemaProcessors, logger }) => {
       );
     });
     const render = () => {
-      return Promise.all(renderers.map(renderer => renderer())).then(metas =>
+      return executeSequentially(renderers).then(metas =>
```

Comment:
Why does this need to be sequential too? I understand that the items within a proposition need to be rendered sequentially, but how about different propositions? I didn't think their order was important.

Reply:
in target, activities can be prioritized which results in the edge response propositions being ordered according to that priority.  in this case, order matters.